extensions/resource-deployment/src/ui/modelViewUtils.ts

```diff
@@ -806,7 +806,11 @@ async function processAzureAccountField(context: AzureAccountFieldContext): Prom
 		await vscode.commands.executeCommand('workbench.actions.modal.linkedAccount');
 		await populateAzureAccounts();
 	}));
-	await populateAzureAccounts();
+
+	// populate the values in a different batch as the initialization to avoid the issue that the account list is empty even though the values are correctly.
```

Comment:
But why is this happening? Is this that annoying race condition issue with the ModelView components again?

Reply:
yes, request of setting actual dropdown values got executed first and then the request of setting the values to empty list because the all came in the same batch.

Reply:
Odd that it's still having this issue - normally it happens if the data is set immediately after creation. And given that this is already in an async functionWhat if instead you fetched the accounts before creating the dropdown - and then just initialized the dropdown with those accounts? I just don't know if we can safely assume this is always going to fix the issue as it is. 

(realistically we need to fix that race condition bug - but that's a lot larger task so I guess doing something temporary like this in the short term is ok)